server/analytics.cc

```diff
@@ -48,11 +48,15 @@ bool iterateDataset(const SelectExpression & select,
                     ssize_t limit,
                     std::function<bool (const Json::Value &)> onProgress)
 {
-    return BoundSelectQuery(select, from, alias, when, where, orderBy, calc)
-        .execute(processor, offset, limit, onProgress);
+    BoundSelectQuery query(select, from, alias, when, where, orderBy, calc);
+
+    query.execute(processor, offset, limit, onProgress);
+
+    return query.selectInfo;
+
 }
 
-bool iterateDatasetExpr(const SelectExpression & select,
```

Comment:
Again, these bools have been removed several times and we've had to put them back each time.  We know from experience that we need them in.